crates/fluvio-extension-common/src/common/time.rs

```diff
@@ -0,0 +1,141 @@
+use chrono::{DateTime, Utc};
+
+const SIMPLE_DATE_FORMAT: &str = "%Y/%m/%d";
+const EXACT_FORMAT: &str = "%Y-%m-%d %H:%M:%S %Z";
+
+pub struct TimeElapsedFormatter {
+    bot: DateTime<Utc>,
+}
+
+impl Default for TimeElapsedFormatter {
+    fn default() -> Self {
+        Self { bot: Utc::now() }
+    }
+}
+
+impl TimeElapsedFormatter {
+    pub fn new(bot: DateTime<Utc>) -> Self {
+        Self { bot }
+    }
+
+    pub fn time_since_str(&self, date: DateTime<Utc>, exact: bool) -> String {
```

Comment:
should add docs on rule

Reply:
Here I have added docs: [e96128c](https://github.com/infinyon/fluvio/pull/4203/commits/e96128c58f5e514fae653130fb9eb4c5486f06ad), I will use such statements in the docs as guide for free-form expressions.